visma/matrix/structure.py

```diff
@@ -83,6 +80,22 @@ def dimension(self):
         self.dim[0] = len(self.value)
         self.dim[1] = len(self.value[0])
 
+    def transposeMat(mat):
```

Comment:
Since you have defined it as a class method you can pass the object itself by using `self` instead of `mat` in the function argument.
```suggestion
    def transposeMat(self):
```